docs/reference.cue

```diff
@@ -1 +1,36 @@
 package metadata
+
+#Fields: [Name=string]: #Fields | _
+
+remap: {
+  errors: [Name=string]: {
+    description: string
+    name: Name
+  }
+
+  functions: [Name=string]: {
+    arguments: [
+      ...{
+        required: bool,
+
+        if !required {
```

Comment:
All arguments can be written as keyword arguments, not just optional ones (and vice versa, all arguments can also be written as positional arguments, including optional ones).